### PR TITLE
Add jxrlib-0.2.4

### PR DESCRIPTION
**Pull request by shuryanc**
Add jxrlib, which will be for a future package.

### CODE REVIEW

File: components/library/jxrlib/patches/01-Makefile.patch [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
++	ginstall $(DIR_BUILD)/$(ENCAPP) $(DIR_BUILD)/$(DECAPP) $(DIR_INSTALL)/usr/bin
++	ginstall -m 644 $(DIR_SRC)/common/include/*.h $(DIR_INSTALL)/usr/include/libjxr/common
++	ginstall -m 644 $(DIR_SRC)/image/x86/*.h $(DIR_INSTALL)/usr/include/libjxr/image/x86
++	ginstall -m 644 $(DIR_SRC)/$(DIR_SYS)/*.h $(DIR_INSTALL)/usr/include/libjxr/image
++	ginstall -m 644 $(DIR_SRC)/$(DIR_ENC)/*.h $(DIR_INSTALL)/usr/include/libjxr/image
++	ginstall -m 644 $(DIR_SRC)/$(DIR_DEC)/*.h $(DIR_INSTALL)/usr/include/libjxr/image
++	ginstall -m 644 $(DIR_SRC)/$(DIR_GLUE)/*.h $(DIR_INSTALL)/usr/include/libjxr/glue
++	ginstall -m 644 $(DIR_SRC)/$(DIR_TEST)/*.h $(DIR_INSTALL)/usr/include/libjxr/test
++	ginstall -m 644 doc/* $(DIR_INSTALL)/usr/share/doc/jxr-$(JXR_VERSION)
+
```
Comment (AndWac) [inline components/library/jxrlib/patches/01-Makefile.patch:55]: Maybe you should try to use
PATH=$(PATH.gnu)
in the Makefile. It should let the build system find the gnu install version.

**Comment by shuryanc** [comment]
Yes, this is better.

File: components/library/jxrlib/Makefile [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+COMPONENT_PROJECT_URL=	https://github.com/glencoesoftware/jxrlib
+COMPONENT_FMRI=		library/jxlib
+COMPONENT_CLASSIFICATION=System/Libraries
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).zip
+COMPONENT_ARCHIVE_URL=	https://github.com/glencoesoftware/$(COMPONENT_NAME)/archive/refs/tags/v$(COMPONENT_VERSION).zip
+COMPONENT_ARCHIVE_HASH=	sha256:aef05b75f33551baf03373e13011ca25623632cfe6079b2d0ea32f4535e72b75
+COMPONENT_LICENSE=	GPL
+COMPONENT_LICENSE_FILE=	LICENSE
+
```
Comment (AndWac) [inline components/library/jxrlib/Makefile:32]: The contents of the LICENSE file says BSD 2-clause, here you stated the license is GPL (which version?)!

**Comment by shuryanc** [comment]
From what I know, its a mix of BSD and GPLv2 license.
As the license only mentioned BSD 2, I will change it to BSD2

File: components/library/jxrlib/Makefile [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+COMPONENT_SUMMARY=	Decode and encode JPEG-XR data from Java
+COMPONENT_PROJECT_URL=	https://github.com/glencoesoftware/jxrlib
+COMPONENT_FMRI=		library/jxlib
+COMPONENT_CLASSIFICATION=System/Libraries
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).zip
+COMPONENT_ARCHIVE_URL=	https://github.com/glencoesoftware/$(COMPONENT_NAME)/archive/refs/tags/v$(COMPONENT_VERSION).zip
+COMPONENT_ARCHIVE_HASH=	sha256:aef05b75f33551baf03373e13011ca25623632cfe6079b2d0ea32f4535e72b75
+COMPONENT_LICENSE=	BSD2CLAUSE
+COMPONENT_LICENSE_FILE=	LICENSE
```
Comment (AndWac) [inline components/library/jxrlib/Makefile:31]: Please use BSD 2-Clause as in other packages.

**Pull request merged**